vector/src/main/java/im/vector/activity/VectorRoomInviteMembersActivity.java

```diff
@@ -230,7 +258,7 @@ public boolean onChildClick(ExpandableListView parent, View v, int groupPosition
                         if (mode.equals(VectorRoomCreationActivity.RoomCreationModes.DIRECT_CHAT)) {
 
                             final ParticipantAdapterItem participant = (ParticipantAdapterItem) mAdapter.getChild(groupPosition, childPosition);
-                            DinsicUtils.startDialogue(VectorRoomInviteMembersActivity.this, mSession, participant);
+                            DinsicUtils.startDialog(VectorRoomInviteMembersActivity.this, mSession, participant);
 
                         } else if (mode.equals(VectorRoomCreationActivity.RoomCreationModes.DISCUSSION)) {
                             addParticipantToListToInvite((ParticipantAdapterItem) item);
```

Comment:
We can remove this second check on mMode, and call directly 
`addParticipantToListToInvite((ParticipantAdapterItem) item);`